src/averageAges.js

```diff
@@ -20,6 +20,20 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let filteredMen = people.filter(person => person.sex === 'm');
+
+  if (arguments.length === 2) {
+    filteredMen = filteredMen.filter(person =>
+      Math.ceil(person.died / 100) === century);
+  }
+
+  let average = filteredMen.reduce(function(sum, person) {
+    return (sum + (person.died - person.born));
+  }, 0);
```

Comment:
Get use to write arrow functions for callbacks
```suggestion
  let average = filteredMen.reduce((sum, person) => (
    sum + (person.died - person.born)
  ), 0);
```